### DISCUSSION TITLE
Zeit is now vercel

### DISCUSSION THREAD

**Discussion by raghav4** [vercel/vercel#4176] [General] [answered]
Recently heard this. When I went to the Zeit site. I saw something different.

> Vercel is the optimal workflow for front-end teams.
All-in-one: Static and Jamstack deployment, Serverless Functions, and Global CDN.

Does that mean I won't be able to deploy my **nodejs** applications anymore on Zeit/Vercel? If yes, will the subdomain remain now.sh or has it been changed to Vercel dot something? (I'd have preferred now.sh for its length :| ).

**Comment by coetry** [comment, marked as answer]
We no longer support container workloads like we did in the early days of ZEIT, but everything else remains the same: static frontend workflows + serverless functions. 

Traditional "serverfull" node apps don't work well on our platform and others are better suited for this.

Your existing `now.sh` deployments will continue working, but moving forward we will be adopting a new suffix: `*.vercel.app` 

We cover all of these details as well as the expected change timeline in [this Notion document](https://www.notion.so/vercel/Vercel-Launch-1cab2369ab1c4d8db1987b68708be683)

**Reply by idjevm** [reply]
That's sad to be honest. now.sh was so much better than vercel.app

**Reply by raghav4** [reply]
@idjevm true!

**Reply by raghav4** [reply]
> We no longer support container workloads like we did in the early days of ZEIT, but everything else remains the same: static frontend workflows + serverless functions

Um, does that mean I won't be able to deploy nodejs + express applications anymore on ZEIT?

**Reply by anildugar** [reply]
I tried deploying a sample nodejs+express app which uses @now/node-server package, but the build just doesn't end which i think is no longer supported, but @coetry can confirm whether node server based apps are supported or not.

**Reply by idjevm** [reply]
AFAIK @now/node-server package is no longer being maintained and has been deprecated. You should try using @now/node instead, that should solve your issues.

**Reply by anildugar** [reply]
so you are suggesting to go for a serverless based app rather than an express based server app correct ?

**Reply by mcsdevv** [reply]
That would be correct, whilst technically possible to use a Serverless Function for an Express-based (stateful) app, I would strongly recommend against it.
